### PR TITLE
Swap uart-rs485-board1 and uart-rs485-board2.

### PR DESCRIPTION
**Pull request by floitsch**
No other change except the one with the comment.

### CODE REVIEW

File: tests/hw/esp32/uart-rs485-board2.toit [hunk 1]
```diff
@@ -5,28 +5,49 @@
...
-RTS ::= 22
-TX ::= 23
+/**
+Tests that the uart in rs485-half-duplex can receive data as soon as the
+  RTS bit is cleared.
+
+Setup:
+  Connect pin 23 of board 1 to pin 22 of board 2.
+  Connect pin 22 of board 1 to pin 23 of board 2.
+  Keep pin 16 of board 2 unconnected.
```
Comment (floitsch) [inline tests/hw/esp32/uart-rs485-board2.toit:18]: Changed from "board 1" to "board 2"

File: tests/hw/esp32/uart-rs485-board2.toit [hunk 2]
```diff
@@ -5,28 +5,49 @@
...
+      --rx=rx
       --tx=tx
+      --rts=rts
       --baud-rate=9600
+      --mode=uart.Port.MODE-RS485-HALF-DUPLEX
+
+  task --background::
+    sleep --ms=2000
+    print "Timeout"
+    print "Board1 must be started before board1"
```
Comment (kasperl) [inline tests/hw/esp32/uart-rs485-board2.toit:43]: wat - that's tough

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/hw/esp32/uart-rs485-board2.toit (hunk 2 above)
Comment (floitsch) [reply tests/hw/esp32/uart-rs485-board2.toit:43]: oops.
done.

**Pull request merged**